openapi.yaml

```diff
@@ -4359,7 +4359,8 @@ paths:
       - Linode Instances
       summary: Delete Disk
       description: |
-        Deletes a Disk you have permission to `read_write`.
+        Deletes a Disk you have permission to `read_write`. The Disk cannot be
+        deleted if it is attached to a Linode's running configuration profile.
```

Comment:
this might be clearer as
`The Disk cannot be deleted if it is attached to a running Linode. You can view a Linode's Disks by examining its configuration profile.`

Reply:
On second thought, hold off on the above for a bit; clearing something up.